dataconnect/dataconnect/schema/schema.gql

```diff
@@ -0,0 +1,89 @@
+# Movies
+type Movie
+  # The below parameter values are generated by default with @table, and can be edited manually.
+  @table {
+  # implicitly calls @col to generates a column name. ex: @col(name: "movie_id")
+  # for UUID fields, @default(expr: "uuidV4()") is implicitly called
+  id: UUID!
+  title: String!
+  imageUrl: String!
+  releaseYear: Int
+  genre: String
+  rating: Float
+  description: String
+  tags: [String]
+  # Vectors -- uncomment the line below to enable vector search
+  descriptionEmbedding: Vector @col(size:768)
+}
+
+# Movie Metadata
+# Movie - MovieMetadata is a one-to-one relationship
+type MovieMetadata
+  @table {
+  # @ref creates a field in the current table (MovieMetadata) that holds the primary key of the referenced type
+  # In this case, @ref(fields: "id") is implied
```

Comment:
`@ref(fields: "movieId", references: "id")` is implied.

The expanded schema is:
```
type MovieMetadata @table {
  id: UUID
  movie: Movie! @ref
  movieId: UUID
  director: String
}

```